src/app.routes.js

```diff
@@ -5,26 +5,45 @@ angular.module('app-bootstrap').config([
   '$urlRouterProvider',
   function($stateProvider, $urlRouterProvider) {
     $stateProvider
-      .state('home', {
+      .state('navbar.home', {
```

Comment:
you can do this, or another option is put it inside navbar state and the name just 'home'. It's the same